### ISSUE TITLE
Create a new release?

### ISSUE THREAD

**Issue by royduin** [toopay/bootstrap-markdown#301]
The latest version is 2.10.0 which doesn't include the Dropzone integration.

**Comment by Hansschouten** [comment]
I just wasted an hour with npm install to find out that the 2.10 release has an error in the package.json, fixed more than a year ago: [#242](https://github.com/toopay/bootstrap-markdown/pull/242). A new release would save other trying to install bootstrap-markdown some time.

**Comment by royduin** [comment]
@Hansschouten, see: https://github.com/toopay/bootstrap-markdown/issues/303 how to install the latest version with NPM from Github 👍 if you haven't found it already.. 😉

**Comment by Hansschouten** [comment]
Yes I solved it, `require('bootstrap-markdown/js/bootstrap-markdown');` did the trick. Thank you.

**Comment by toopay** [comment]
I still prepare best strategy to ship latest feature and starts support Bootstrap 4 at the same time. Hopefully this week. 

    On Thursday, August 24, 2017 7:04 PM, Hans Schouten <notifications@github.com> wrote:
 

 Yes I solved it, require('bootstrap-markdown/js/bootstrap-markdown'); did the trick. Thank you.—

**Comment by niknokseyer** [comment]
You need any help on the Bootstrap 4 support? @toopay

**Comment by NicolasCARPi** [comment]
No new release since 2 years and half, or in javascript time about 600 years. Is there a blocking issue? In the meantime bootstrap 4 and fontawesome 5 went out.

@toopay: are you still maintaining this repo? It seems issues are not triaged after mid july 2017 (a year ago). No answer on issues or PRs.

**Comment by toopay** [comment]
@NicolasCARPi Honestly,  I had intentions that I’d have more time to dedicate towards open source projects but reality worked out slightly differently and I’ve had one of the busiest (and best) years of my life.

The other principal factor is truth be told I don’t actively use the project any more. This past year I’ve written more lines of Erlang and Bash than I have Javascript.

That the project has stagnated somewhat has started to bother me greatly. But at least this month, I wouldn't be able to do anything about that. Hopefully next month. :crossed_fingers:

**Comment by toopay** [comment]
@niknokseyer Yes please. If you could make some initial PR for supporting Bootstrap 4, that would be awesome! Thanks

**Comment by NicolasCARPi** [comment]
@toopay Ok thanks for your answer. What you describe is natural in the lifecycle of an open source project and it's totally fine! I think this repo has reached a critical size where it can be maintained by other users. I understand that you are not using this software anymore and hence don't really have the incentive to maintain it properly.

If you find the time to review and merge PRs and create new releases it's great, otherwise you might want to consider "giving the keys" to someone else, motivated and skillfull :).

Cheers,
~Nico
